### PR TITLE
Port tests_bridge.yml to use the same format as the new tests

### PR DESCRIPTION
**Pull request by RMPR**
Addressing #167.

### CODE REVIEW

**Comment by RMPR** [comment]
@tyll asking a review for this.

File: tests/tests_bridge_nm.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      register: NetworkManager_version
+  vars:
+    network_provider_current: nm
+
+- import_playbook: playbooks/tests_bridge.yml
+  when:
+    - ansible_distribution_major_version != '6'
+    # NetworkManager 1.20.0 introduced ethtool settings support
+    # https://github.com/NetworkManager/NetworkManager/blob/master/NEWS
+    - NetworkManager_version.stdout is version('1.20.0', '>=')
```
Comment (tyll) [inline tests/tests_bridge_nm.yml]: This check and the related tasks are not needed, since this test is about bridge support and not ethtools support.

Comment (RMPR) [reply tests/tests_bridge_nm.yml]: I think it's good now.

Comment (RMPR) [reply tests/tests_bridge_nm.yml]: Do I need to remove tests_bridge_other_provider.yml as part of this PR?

Comment (tyll) [reply tests/tests_bridge_nm.yml]: > Do I need to remove tests_bridge_other_provider.yml as part of this PR?

yes, please

Comment (RMPR) [reply tests/tests_bridge_nm.yml]: Done

**Comment by tyll** [comment]
[citest commit:d4199003e9d24b3a81086f1b8d4991b9cb893169]

**Comment by tyll** [comment]
@RMPR the test fails, did you run it on your local machine?

**Comment by RMPR** [comment]
Yes I run the tox tests.

Mar 25, 2020 3:24:13 PM Till Maas <notifications@github.com>:

**Comment by tyll** [comment]
> Yes I run the tox tests.

Tox does not run the integration tests, please take a look at https://github.com/linux-system-roles/network/wiki#improve-the-testing-framework

**Comment by tyll** [comment]
This needs to handle `down-profile.yml` as the CI showed.

**Comment by tyll** [comment]
Followup is in #197

**Pull request closed without merging**